ui/src/testing/stubs/runtime-api-stub.ts

```diff
@@ -8,18 +8,18 @@ import {
   RuntimeStatus,
 } from 'generated/fetch';
 
+import { diskApi } from 'app/services/swagger-fetch-clients';
 import {
   DATAPROC_WORKER_MIN_DISK_SIZE_GB,
   MIN_DISK_SIZE_GB,
-} from 'app/pages/analysis/runtime-panel';
-import { diskApi } from 'app/services/swagger-fetch-clients';
+} from 'app/utils/machines';
 
 import { stubNotImplementedError } from 'testing/stubs/stub-utils';
 
 import { DiskApiStub, stubDisk } from './disk-api-stub';
 
 export const defaultGceConfig = (): GceConfig => ({
-  diskSize: 120,
+  diskSize: MIN_DISK_SIZE_GB + 30,
```

Comment:
Hmm...  I see it's equal to `DATAPROC_WORKER_MIN_DISK_SIZE_GB` but this is not Dataproc so it makes sense for us not to use that const.  But where does the +30 come from?

Reply:
Added a comment here